django_fido/tests/test_views.py

```diff
@@ -50,7 +50,9 @@ def _get_fido2_request(self, challenge, credentials):
         }
         fido2_request = {'publicKey': {
             'rp': rp_data,
-            'user': {'displayName': USER_FULL_NAME, 'id': websafe_encode(bytes(USERNAME, encoding="utf-8")), 'name': USERNAME},
+            'user': {
```

Comment:
Why this change?

Reply:
To simulate behaviour of `Fido2Encoder` it should be identical now.